### PR TITLE
Update knowledge-submissions-past-wikipedia.md

### LINKED ISSUE #59: need recommendations for trusted data sources beyond Wikipedia for Knowledge submissions
For background on this issue, please review this document on [InstructLab knowledge submissions beyond Wikipedia](https://github.com/instructlab/dev-docs/blob/main/docs/knowledge-submissions-past-wikipedia.md). 

There are any number of other domains we may wish to add as approved content providers. Please use this issue to make recommendations for review by legal experts.

### PR DESCRIPTION
**Pull request by lhawthorn**
Added new approved Knowledge submission data sources, updated status for several.

Updated documentation with new process to take in requested knowledge sources to be open a PR against this devdoc.

Related to issue #59 which should be closed once this PR is reviewed and merged.

Closes #59

### CODE REVIEW

**Comment by jjasghar** [comment]
I helped out yoru linting issues, the spell check doesn't like a bunch of the words...i guess we'll need to add them to the spell checker now:
```
Cynefin
Knowlege
OpenStax
PLOS
Standup
butvis
io
```

File: docs/knowledge-submissions-past-wikipedia.md
Comment (markstur) [inline docs/knowledge-submissions-past-wikipedia.md:29]: s/Knowlege/Knowledge/

File: docs/knowledge-submissions-past-wikipedia.md
Comment (markstur) [inline docs/knowledge-submissions-past-wikipedia.md:57]: typo s/butvis/but is/

**Comment by lhawthorn** [comment]
LH is unaware of the existence of a Mergify dashboard and will get educated when maintainer team is online.

**Comment by lhawthorn** [comment]
@mergifyio rebase

**Review by markstur — APPROVED** [review]
LGTM

File: docs/knowledge-submissions-past-wikipedia.md
Comment (bjhargrave) [inline docs/knowledge-submissions-past-wikipedia.md:27]:
```suggestion
- `reviewed - manually verify`: Legal team has reviewed this domain and while much of its source material meets our open licensing criteria, not all of it does. Each submission from this source must be manually verified to actually be under an appropriate content license or definitively in the public domain.
```

File: docs/knowledge-submissions-past-wikipedia.md
Comment (bjhargrave) [inline docs/knowledge-submissions-past-wikipedia.md:60]:
```suggestion
- [Approved sources][approved]
```

File: docs/knowledge-submissions-past-wikipedia.md
Comment (bjhargrave) [inline docs/knowledge-submissions-past-wikipedia.md:61]:
```suggestion
- [Rejected sources][avoided]
```

**Review by bjhargrave — CHANGES_REQUESTED** [review]

**Review by nathan-weinberg — APPROVED** [review]
Pending @bjhargrave's comments this LGTM

File: docs/knowledge-submissions-past-wikipedia.md
Comment (hickeyma) [reply docs/knowledge-submissions-past-wikipedia.md:27]: Updated

**Comment by bjhargrave** [comment]
@hickeyma Your fix commit is not pushed to this PR. It is just in your fork.

**Review by hickeyma — APPROVED** [review]
LGTM thanks @lhawthorn  for the update of the updates. 

@bjhargrave Added your suggestions. Do you mind doing a quick review?

**Comment by hickeyma** [comment]
Shoot, I see that @bjhargrave. Let me try and fix that.

**Comment by hickeyma** [comment]
Thanks @bjhargrave for the help with the initial commit. Ready for review now.

**Review by bjhargrave — APPROVED** [review]

**Pull request merged**
